components/gemini/browser/gemini_service.h

```diff
@@ -0,0 +1,162 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_GEMINI_BROWSER_GEMINI_SERVICE_H_
+#define BRAVE_COMPONENTS_GEMINI_BROWSER_GEMINI_SERVICE_H_
+
+#include <list>
```

Comment:
are all these includes used? It doesn't seem like it

Reply:
it's used here

Reply:
it's used here: https://github.com/brave/brave-core/pull/6062/files#diff-2f1a6303ff1481ad0ef7c6fccf93fd06R100